app/config/services/platform.xml

```diff
@@ -222,7 +225,9 @@
         </service>
 
         <!-- Events -->
-        <service id="app.event.factory" class="AppBundle\Event\EventFactory"/>
+        <service id="app.event.factory" class="AppBundle\Event\EventFactory">
+            <argument type="service" id="AppBundle\Referent\ReferentTagManager"/>
+        </service>
```

Comment:
It's a suggestion but why not putting this service in services.xml (as public=true because it's needed in 2 fixtures loader).